api/getPageContentByPathname/index.ts

```diff
@@ -0,0 +1,27 @@
+import { CONTENTFUL_API_URL, headers, HttpRequestMethod } from '../constants'
```

Comment:
`HttpRequestMethod` is a type so might be good to explicitly import it this way. Thinking about it maybe this type should be in the `types.ts` rather than the `constants.ts` as well.